addrs/provider_config_test.go

```diff
@@ -16,143 +16,147 @@ func TestParseAbsProviderConfig(t *testing.T) {
 		WantDiag string
 	}{
 		{
-			`provider.aws`,
+			`provider.["registry.terraform.io/hashicorp/aws"]`,
```

Comment:
This is the part to pay attention to! There are plenty of optional variations on the syntax - I started with this one because it was both easy to parse as a human, and easy to parse as HCL with minimal changes.